pyxem/signals/electron_diffraction.py

```diff
@@ -369,39 +371,30 @@ def remove_deadpixels(self,
                         deadvalue=deadvalue,
                         inplace=inplace)
 
-    def get_radial_profile(self, centers=None):
+    def get_radial_profile(self,cython=False):
```

Comment:
*args & **kwargs here? Also, pass in_place as keyword to this function for use in map later?